### ISSUE TITLE
Description of exp() wrongly called it "pow"

### ISSUE THREAD

**Issue by GeForceLegend** [nolanderc/glsl_analyzer#22]
Line 7215 in spec/spec.json
```json
    {
      "return_type": "genType",
      "name": "exp",
      "parameters": [
        {
          "type": "genType",
          "name": "x"
        }
      ],
      "description": [
        "`pow` returns the natural exponentiation of _x_. i.e., `e**x`."  // <- Here
      ],
      "versions": [
        110,
        120,
        130,
        140,
        150,
        330,
        400,
        410,
        420,
        430,
        440,
        450
      ]
    },
```

**Comment by nolanderc** [comment]
Nice catch!

Looks like an error upstream: https://github.com/BSVino/docs.gl/blob/080c215e49bc57289e2490008adcf990a5244312/sl4/exp.xhtml#L43

**Comment by nolanderc** [comment]
This was fixed in upstream, and now part of v1.1.5
